packages/backend/src/app/webhooks/webhook-simulation-controller.ts

```diff
@@ -0,0 +1,47 @@
+import { FastifyPluginAsync, FastifyRequest } from "fastify";
+import { FlowId, WebhookUrlParams } from "@activepieces/shared";
+import { StatusCodes } from "http-status-codes";
+import { logger } from "../helper/logger";
+import { webhookService } from "./webhook-service";
+
+export const webhookSimulationController: FastifyPluginAsync = async (app) => {
+    app.all(
+        "/:flowId",
+        {
+            schema: {
+                params: WebhookUrlParams,
+            },
+        },
+        async (request: FastifyRequest<{ Params: WebhookUrlParams }>, reply) => {
+            await handler(request, request.params.flowId);
+            await reply.status(StatusCodes.OK).send();
+        }
+    );
+
+    app.all(
```

Comment:
Let's remove this endpoint, It's not needed.

Reply:
fixed in 9dacacd8